src/components/MapView/Layers/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { Fragment } from 'react';
+import { Source, Layer } from 'react-mapbox-gl';
+import { useSelector } from 'react-redux';
+import { chain } from 'lodash';
+
+import { selectlayers } from '../../../context/filters/filtersSlice';
+
+const wmsParams = chain({
+  version: '1.1.1',
+  request: 'GetMap',
+  format: 'image/png',
+  transparent: true,
+  exceptions: 'application/vnd.ogc.se_inimage',
+  bboxsr: 3857,
+  imagesr: 3857,
+  width: 256,
+  height: 256,
+  srs: 'EPSG:3857',
+  bbox: '{bbox-epsg-3857}',
+})
+  .map((value, key) => `${key}=${value}`)
+  .join('&')
+  .value();
```

Comment:
This seems like reinventing the wheel, and could be dangerous without escaping URI components.

You could use Node's [url library](https://www.npmjs.com/package/url)